manifests/controllerconfig.crd.yaml

```diff
@@ -417,7 +416,7 @@ spec:
                               resourceGroupID:
                                 description: resourceGroupID is the ID of the resource group for the cluster.
                                 type: string
-                                pattern: ^rg-[0-9A-Za-z]+$
+                                pattern: ^(rg-[0-9A-Za-z]+)?$
```

Comment:
we need to check if it will not be replaced by auto-generated code or need to updated here: https://github.com/openshift/machine-config-operator/blob/master/pkg/operator/assets/bindata.go#L586